src/main/java/com/faforever/client/game/GameService.java

```diff
@@ -198,12 +204,12 @@ public GameService(ClientProperties clientProperties, FafService fafService,
     JavaFxUtil.attachListToMap(games, uidToGameInfoBean);
   }
 
-  
+
```

Comment:
no idea where all these empty lines come from, but they should be removed.

Reply:
They are not empty the ide only removed the whitespsaces...

Reply:
I checked in my IDE and they are